packages/kbn-es-query/src/es_query/types.ts

```diff
@@ -58,6 +58,10 @@ export type DataViewFieldBase = {
    * ES field types as strings array.
    */
   esTypes?: string[];
+  /**
+   * Description of field type conflicts across indices
+   */
+  conflictDescriptions?: Record<string, string[]>;
```

Comment:
I'd prefer this remain unchanged. I'm kind of curious how fields are being loaded where this would be helpful.

Reply:
@e40pud It still might be worth having a conversation about how the field list is being loaded to make sure the data views api is doing what it can in this circumstance.

Reply:
@elastic/security-solution-platform - @e40pud and I had a conversation about how fields are being loaded and it seems like the data views api could be doing more when dealing with unmapped fields. My goal is that you should be able to get the fields you need and related data in a single call. Please reach out to me if you'd like to discuss this further.

Reply:
Just for a reference, [here is the place](https://github.com/elastic/kibana/pull/149149/files#diff-bb501e7a13c4b40ac43d132631601dc5cf7090b3ce6179379cd05ca5a1bfd3c0R100) where I do extra call to get unmapped and conflicts information in data views case.